src/environment.rs

```diff
@@ -159,6 +161,39 @@ fn create_history_file(environment_dir: &Path) -> miette::Result<()> {
     Ok(())
 }
 
+#[derive(Serialize, Deserialize)]
+pub(crate) struct EnvironmentFile {
+    pub(crate) manifest_path: PathBuf,
+    pub(crate) environment_name: String,
+    pub(crate) pixi_version: String,
+}
+/// Write information about the environment to a file in the environment directory.
+/// This can be useful for other tools that only know the environment directory to find the original project.
+pub fn write_environment_file(
+    environment_dir: &Path,
+    env_file: EnvironmentFile,
+) -> miette::Result<PathBuf> {
+    let path = environment_dir
+        .join("conda-meta")
+        .join(consts::ENVIRONMENT_FILE_NAME);
+
+    let parent = path
+        .parent()
+        .expect("There should already be a conda-meta folder");
+
+    if !parent.exists() {
+        std::fs::create_dir_all(parent).into_diagnostic()?
+    }
```

Comment:
```suggestion 
std::fs::create_dir_all(parent).into_diagnostic()?
```

No need to check if the parent exists, this function already does that.